### DISCUSSION TITLE
Mozilla Hub Unserviceable After AWS Database Master Account Password Change (502 Error)

### DISCUSSION THREAD

**Discussion by lln0302** [mozilla/hubs#6066] [Hubs Cloud Questions]
I wanted to check AWS's postgresql database, so I tried to connect to dbbeaver. I didn't do the existing settings, so I didn't know the master db password of aws rds, so I randomly changed it, but since then, I've been unable to connect to the Mozilla Hub. (502 error) What should I do in a situation like this?
1. Should I find my old password and change it? Can I contact the aws customer center to find the password?

2. Should I change the connection settings in the code or admin of the Mozilla Hub? (admin is possible because the Mozilla Hub itself doesn't connect?)

I look forward to hearing from you. Thank you.

**Comment by pattersonbl2** [comment]
You will have to ssh into the ec2 instant.  once you are connected run  `sudo su - root` then `vim /hab/svc/reticulum/config/config.toml`.   you will need the [ret."Elixir.Ret.Repo"] information to find the password.  You could also look in the AWS parameter store for the old password as well.
